### PR TITLE
Noarch packaging

### PR DESCRIPTION
**Pull request by borupdaniel**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,15 +11,13 @@ source:
...
+    python
   number: {{ build_num }}
   skip_compile_pyc:
     - "include/PyFI/*.py"
-  skip: true  # [win]
-  skip: true  # [py2k]
 
 requirements:
-  host:
-    - python {{ python }}
```
Comment (grlee77) [inline recipe/meta.yaml:22]: I think you need to keep:
```
  host:
    - python
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -28,15 +26,13 @@ requirements:
     - pillow
     - psutil
     - pyopengl
-    - pyqt >=5.6
-    - {{ pin_compatible('python') }}
```
Comment (grlee77) [inline recipe/meta.yaml:32]: keep this as just ``python`` without the pin_compatible logic

Comment (borupdaniel) [reply recipe/meta.yaml:32]: I think that's taken care of in the change already, isn't it?

**Comment by borupdaniel** [comment]
I think this is ready to go, but with a few other things in the pipeline the next few days and the holiday next week, I am going to hold off on accepting this merge until the first week of December.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by borupdaniel** [comment]
@ocefpaf thanks for the help! Are the jinja things like `{{ sha256 }}` not allowed in `noarch` packages or is that just a style convention?

**Comment by ocefpaf** [comment]
We should use Jinja only for variables that are used in multiple places or the version.

**Comment by borupdaniel** [comment]
@ocefpaf - could you please tag this build (`1.1.5-py_1`) as `broken` as it leads to a circular dependency on `gpi_core`.

**Comment by ocefpaf** [comment]
> @ocefpaf - could you please tag this build (`1.1.5-py_1`) as `broken` as it leads to a circular dependency on `gpi_core`.

Sure. See https://anaconda.org/conda-forge/gpi/files?version=1.1.5

BTW, ping `@conda-forge/core` next time for this kind of operation b/c I may not be available.

**Comment by borupdaniel** [comment]
> BTW, ping `@conda-forge/core` next time for this kind of operation b/c I may not be available.

Will do, thanks!

**Comment by mbargull** [comment]
As is, it is contradictory to use `noarch` when the `build.sh` has system dependent conditional code like `if [ "$(uname)" ...`. Meaning, this shouldn't be `noarch` unless the build is changed to have consistent output supporting all platforms.

**Pull request merged**